lib/videoroom/meeting.ex

```diff
@@ -142,23 +141,16 @@ defmodule Videoroom.Meeting do
   end
 
   @impl true
-  # Handle specific notifications for the current room
-  def handle_info({:jellyfish, %type{room_id: id} = notification}, %{room_id: id} = state)
-      when type in [PeerConnected, PeerDisconnected, PeerCrashed, RoomCrashed] do
+  def handle_info({:jellyfish, notification}, state) do
     Logger.info("jellyfish notification: #{inspect(notification)}")
     handle_notification(notification, state)
   end
 
-  # Handle all other notifications, including ones from the other rooms
-  def handle_info({:jellyfish, notification}, state) do
-    Logger.debug("jellyfish notification: #{inspect(notification)}")
+  def handle_info({:DOWN, _ref, :process, _notifier_pid, _reason}, state) do
+    Logger.error("Connection to jellyfish closed!")
```

Comment:
I would leave `Notifier` as it is, but let's remove this `handle_info` (I believe that this process doesn't monitor the `Notifier` process because why would it?). @roznawsk